app/src/androidTest/java/org/mozilla/focus/activity/URLCompletionTest.java

```diff
@@ -204,22 +230,21 @@ private static void checkACOff(String url) throws UiObjectNotFoundException {
     }
 
     // Check autocompletion is turned on
-    private static void checkACOn(String url) throws UiObjectNotFoundException {
+    private void checkACOn(String url) throws UiObjectNotFoundException {
         TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime);
-        TestHelper.inlineAutocompleteEditText.setText(url.substring(0, 2));
+        TestHelper.inlineAutocompleteEditText.setText(url.substring(0, 1));
         TestHelper.hint.waitForExists(waitingTime);
         assertTrue (TestHelper.inlineAutocompleteEditText.getText().equals(url));
         TestHelper.cleartextField.click();
     }
 
-    private static void removeACSite() {
-        DataInteraction CustomURLRow = onData(anything())
-                .inAdapterView(allOf(withId(android.R.id.list),
-                        childAtPosition(
-                                withId(android.R.id.list_container),
-                                0)))
-                .atPosition(4);
-        CustomURLRow.perform(click());
+    private void removeACSite() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            CustomURLRow.perform(click());
+        }
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 9 should be on the same line as the next part of a multi\-block statement \(one that directly contains multiple blocks: if/else\-if/else, do/while or try/catch/finally\)\.